js/components/graph/BoolGroup.js

```diff
@@ -18,8 +18,7 @@ export default class BoolGroup extends React.Component {
    */
   reset = () => {
     this.props.boolsJSON.forEach(boolJSON => {
-      var bool = this[boolJSON.id_]
-      bool.setState({ status: "inactive" })
+      this[boolJSON.status] = "inactive"
```

Comment:
Now that the bool status is stored in the `Graph` component, it is no longer necessary to define a `reset` method in this component. Instead, you can modify the `Graph.reset` method to change the status of each bool node.